generate-templates.yaml

```diff
@@ -232,6 +232,36 @@
     vars:
       osinfoname: win2k12r2
 
+  - name: Generate Windows server 2012 R2 templates
```

Comment:
Change `Generate Windows server templates` to `Generate deprecated Windows server templates` in line 225

Reply:
Done